sdcontent/INSTRUCTIONS-README.txt

```diff
@@ -14,21 +14,21 @@ NOOBS INSTALLATION INSTRUCTIONS
       iv. Check that the SD card you inserted matches the one selected by the Tool
       v. Click the "Format" button
    c. Linux
-      i. We recommend using gparted (or the command line version parted)
-      ii. Format the entire disk as FAT32
+      i. We recommend using gparted (or the command line version, parted)
+      ii. Format the entire card as FAT32
    d. Cards over 32GB
-      i. Follow the instructions on https://www.raspberrypi.org/documentation/installation/sdxc_formatting.md
+      i. Follow the instructions at https://www.raspberrypi.org/documentation/installation/sdxc_formatting.md
 3. Extract the files contained in this NOOBS zip file.
-4. Copy the extracted files onto the SD card that you just formatted so that this file is at the root directory of the SD card. Please note that in some cases it may extract the files into a folder, if this is the case then please copy across the files from inside the folder rather than the folder itself.
-5. Insert the SD card into your Pi and connect the power supply.
+4. Copy the extracted files onto the SD card that you just formatted. The files must be in the root directory of the SD card, not in a subdirectory within this. If the files are in a subdirectory on the SD card, then you will not be able to start a Raspberry Pi from the SD card.

```

Comment:
"start a Raspberry Pi" -> "boot a Raspberry Pi" ? :shrug:

Reply:
Hopefully this text won't cause users to think they've done something wrong when they see the `os` subdirectory?

Reply:
I've removed the bit about the Pi not booting if there are files in a subdirectory, as I think that was potentially problematic, as you said.